contracts/RelayHub.sol

```diff
@@ -191,40 +191,20 @@ contract RelayHub is IRelayHub {
             return uint256(PreconditionCheck.WrongNonce);
         }
 
-        bytes memory rawTx = abi.encodeWithSelector(to.acceptRelayedCall.selector,
-            relay, from, encodedFunction, gasPrice, transactionFee, approval);
+        bytes memory encodedTx = abi.encodeWithSelector(to.acceptRelayedCall.selector,
+            relay, from, encodedFunction, gasPrice, transactionFee, approval
+        );
 
-        (bool success, uint256 accept) = staticCallWithMaxGas(address(to), acceptRelayedCallMaxGas, rawTx);
+        (bool success, bytes memory returndata) = address(to).staticcall.gas(acceptRelayedCallMaxGas)(encodedTx);
```

Comment:
see the circleCI build failure: it claims the method can't be  `view`....
Could be that we hit another solc problem.

Reply:
That is exactly the bug that was fixed in 0.5.10 (see the link in the first comment) - Docker doesn't seem to be running the Dockerfile on Circle, I think there's an issue with the cache not being invalidated.

Reply:
@drortirosh I'm not extremely familiar with CircleCI and Docker, but it seems to me like the CI build is not actually running the Dockerfile commands, but rather downloading the published image (`- image: tabookey/eth-tools:solc-5.5b`). Is this so, or is there a step I'm missing somewhere>